src/utils/searchHelper.js

```diff
@@ -101,18 +101,33 @@ export function newSearch() {
 function buildSearchScenesParams(gridCodeToSearchIn) {
   const _selectedCollection = store.getState().mainSlice.selectedCollectionData
   const bbox = buildUrlParamFromBBOX()
-  const dateTimeRange = convertDateForURL(
+  const _dateTimeRange = convertDateForURL(
     store.getState().mainSlice.searchDateRangeValue
   )
   const limit = VITE_API_MAX_ITEMS || 200
   const collections = _selectedCollection.id
-
-  const searchParams = new Map([
-    ['bbox', bbox],
-    ['datetime', dateTimeRange],
-    ['limit', limit],
-    ['collections', collections]
-  ])
+  const _searchGeojsonBoundary =
+    store.getState().mainSlice.searchGeojsonBoundary
+
+  let searchParams
```

Comment:
```suggestion
  let searchParams = new Map([
      ['datetime', _dateTimeRange],
      ['limit', limit],
      ['collections', collections]
    ])
```

and then add intersects or bbox to the Map, instead of duplicating these parameters